package.json

```diff
@@ -1,40 +1,40 @@
 {
-  "name": "parceltest",
```

Comment:
The changes to this file look like just formatting, in that case can you keep the file as is? We'll make formatting changes in a different PR if needed

Reply:
ok, it's my VS code `Tab Size` preferences set to `4 spaces`

Reply:
restored